new evaluations.html

```diff
@@ -2,9 +2,15 @@
 <html lang="en">
 <head>
         <src >
+<<<<<<< Updated upstream
```

Comment:
this is a git conflict that has not been managed and it will break your new evaluations.html visuals